css/_chat.scss

```diff
@@ -295,7 +295,7 @@
     max-height: 0;
     overflow: hidden;
     position: absolute;
-    width: $sidebarWidth;
+    max-width: $sidebarWidth;
```

Comment:
not sure that it will overwride `styles` attribute for the `SmileysPanel` at all possible browsers. I suggest to add. To make it more priority. Not sure.
```suggestion
    max-width: $sidebarWidth !important;
```

Reply:
I'm just afraid that attribute `style` and it's property `width` have more priority in some browsers. I may be wrong.

Reply:
@mike-gusev-ae thanks for chat conversation. Looks like all is fine with that.